contracts/solidity/.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
```

Comment:
mentioned this on discord but just not to forget, this file can be removed

Reply:
Removed.